packages/completer/style/base.css

```diff
@@ -15,20 +16,34 @@
   background: var(--jp-layout-color1);
   color: var(--jp-content-font-color1);
   border: var(--jp-border-width) solid var(--jp-border-color1);
-  list-style-type: none;
-  overflow-y: scroll;
-  overflow-x: auto;
   padding: 0;
+  display: flex;
+  flex-direction: row;
   /* Position the completer relative to the text editor, align the '.' */
   margin: 4px 0 0 -30px;
+  z-index: 10001;
+}
+
+.jp-Completer-docpanel {
+  border-left: var(--jp-border-width) solid var(--jp-border-color1);
+  /* flex-shrink: 10001; /* TODO replace this with a more general solution */
+  width: 300px; /* TODO maybe make this more flexible? */
+  overflow: scroll;
```

Comment:
In the case that the docpanel is against the window, is the content horizontally scrollable as well?

Reply:
I've changed the behavior for window-edges. The completer box now gets pushed left to accommodate the docpanel. I'll update the screenshots to reflect this.